lib/bootloader_pvm.pm

```diff
@@ -131,6 +131,10 @@ sub enter_netboot_parameters {
     type_string_slow remote_install_bootmenu_params;
     type_string_slow " fips=1" if (get_var('FIPS_INSTALLATION'));
     type_string_slow " UPGRADE=1" if (get_var('UPGRADE'));
+
```

Comment:
in specific_bootmenu_params we need to handle AGAMA.AUTO for unattended.

Reply:
Step by step, we need to validate the first scenario then we can jump to the other ones. I will take a look thanks